app/flatpak-builtins-create-usb.c

```diff
@@ -120,6 +120,9 @@ add_related (GHashTable   *all_refs,
   if (deploy_data == NULL)
     return FALSE;
 
+  if (flatpak_deploy_data_has_subpaths (deploy_data))
+    return flatpak_fail (error, _("Related ref '%s' is only partially installed"), ref);
```

Comment:
Why do we want to fail? If you're offline and have your language, the person you're sharing it with probably has the same language right

Reply:
Hmm, its *possible* you want it. But it is more likely you're not even aware of this happening, which will cause you to generate a usb repo that may break for users in weird ways. I.e. the pull would just stop in the middle with some "missing object 123abc..." error.

Maybe we could have a switch that turns this into a warning instead.

Reply:
Yeah, better to have a warning and ideally a more useful error message on the install side than "missing object".